modules/core/shared/src/main/scala/gem/enum/KeywordName.scala

```diff
@@ -727,10 +727,23 @@ object KeywordName {
   /** @group Constructors */ case object MJD_OBS extends KeywordName("MJD_OBS", "MJD_OBS")
   /** @group Constructors */ case object APERTURE extends KeywordName("APERTURE", "APERTURE")
   /** @group Constructors */ case object FILTER extends KeywordName("FILTER", "FILTER")
+  /** @group Constructors */ case object AOFREQ extends KeywordName("AOFREQ", "AOFREQ")
```

Comment:
This just seems to me like a bad idea, with all the repeated strings asking for trouble. Couldn't we have an intermediate class like:

```
sealed abstract class KeywordTagIsName(tag: String) extends KeywordName(tag, tag)
```

and use that when the `tag` and the `name` are the same?

Reply:
`KeywordName` is generated code. I suppose we could do as you suggest and have the few cases where the `tag` and `name` are different but it would complicate a lot the code generation part